delft/eris.nix

```diff
@@ -40,20 +40,19 @@ in {
 
   services.nginx = {
     enable = true;
-    virtualHosts."status.nixos.org" = {
+    virtualHosts."monitoring.nixos.org" = {
       enableACME = true;
       forceSSL = true;
-      root = ./eris/status-page;
```

Comment:
Looks good, but I think nginx will want us to define a root or a / location. I wonder what it takes to make the / a redirect?

Reply:
Oh right, I forgot the redirect.